tpl/template_embedded.go

```diff
@@ -233,7 +233,7 @@ func (t *GoHTMLTemplate) EmbedTemplates() {
 {{ end }}{{ end }}
 
 <!-- Output all taxonomies as schema.org keywords -->
-<meta itemprop="keywords" content="{{ range $plural, $terms := .Site.Taxonomies }}{{ range $term, $val := $terms }}{{ printf "%s," $term }}{{ end }}{{ end }}" />
+<meta itemprop="keywords" content="{{ if .IsPage}}{{ range $index, $tag := .Params.tags }}{{ $tag }},{{ end }}{{ else }}{{ range $plural, $terms := .Site.Taxonomies }}{{ range $term, $val := $terms }}{{ printf "%s," $term }}{{ end }}{{ end }}{{ end }}" />
```

Comment:
> What use case am I missing?

The case where an schema object is repeated in a list and the partial is applied multiple times.

<img width="707" alt="screen shot 2017-04-09 at 8 10 46 pm" src="https://cloud.githubusercontent.com/assets/440298/24837187/ad972e12-1d60-11e7-99d5-dead251fd23e.png">

You can dig into it [via the validator](https://search.google.com/structured-data/testing-tool/u/0/#url=https%3A%2F%2Fhackcabin.com) if you like.

The following schema _Things_ support the attributes (`dateModified`, `datePublished`, `keywords`) provided in this internal template:

- https://schema.org/CreativeWork
- http://schema.org/BlogPosting (inherits from CreativeWork)

If we output a list of site keywords we lose the ability reuse this partial more than once per page (which is a major shortcoming of Jekyll SEO Tag IMO.

Does that make sense?